client/src/pages/UserAdmin.js

```diff
@@ -0,0 +1,318 @@
+    const userEmail = props.userToEdit.email;
+    const userProjects = props.userManagedProjects || [];
+
+    //Processing
+    const cancelEdit = () => {
+        props.handleFormCancel();
+    }
+
+    // add user projects to state
+    useEffect(() => {
+        props.handleAddProject(props.userToEdit.managedProjects);
+    }, []);
+
+    // Prepare user projects for display by connecting the ID with the project name
+    function prepareUserProjects (userProjects, activeProjects ) {
+        let res = activeProjects.filter(item => userProjects.includes(item[0]));
+        return res;
+    } 
+
+    const userProjectsToDisplay = prepareUserProjects (userProjects, props.activeProjects);
+
+    return (
+        <div>
+            <div className="ua-row">
+                <div className="user-display-column-left">
+                    Name: 
+                </div>
+                <div className="user-display-column-right">
+                    {userName} 
+                </div>
+            </div>
+            <div className="ua-row">
+                <div className="user-display-column-left">
+                    Email: 
+                </div>
+                <div className="user-display-column-right">
+                    {userEmail}
+                </div>
+            </div>
+            <div className="ua-row">
+                <div className="user-display-column-left">
+                    Projects: 
+                </div>
+                <div className="user-display-column-right">
+                    <ul className="project-list">    
+                        {userProjectsToDisplay.map((result,index) => {
+                        return (
+                            <li key={index}>{result[1]}
+                            <button className="button-remove" onClick={() => props.handleRemoveProject(result[0])}>(remove)</button>
+                            </li>
+                        )})}
+                    </ul>
+                </div>
+            </div>
+            <div>
+                {<form onSubmit={props.handleFormSubmit}>
+                    <select 
+                        className="project-select"
+                        value={props.projectValue} 
+                        onChange={handleChange}>
+                            <option  value='default'>Select a project..</option>
+                            {props.activeProjects.map((result,index) => {
+                            return (
+                                <option key={index} value={result[0]}>{result[1]}</option>
+                            )})}
+                        </select>
+                        <button className="button-add" type="submit">Add project</button>
+                </form>}
+                <div><button className="button-back" onClick={cancelEdit}>Back to search</button></div>
+            </div>
+        </div>
+    )
+};
+
+
+// Export UserAdmin
+export default UserAdmin;
```

Comment:
please add an empty line at the end of the file

Reply:
done